### ISSUE TITLE
Add parameters to pre-defined actions

### ISSUE THREAD

**Issue by xaviml** [xaviml/controllerx#78]
# Feature Request

### Is your feature request related to a problem?

It is frustrating to create a `CallServiceController` because you want to use a pre-defined action with different parameters. In this task, I will be using `sync` predefined action as an example.

`sync` allows to set a light to maximum brightness and color_temp to 300mired (or 3700K). If you want to set the `sync` pre-defined action to an 80% brightness and 500mired, as of now you cannot do it. The only work around to this is to use `CallServiceController` and all the services yourself with the parameters you want.

### Describe the solution / feature you'd like

What I would like is to set parameters to the pre-defined actions such as `sync`. This is a configuration that uses `sync` action in ControllerX v2.6.0:
```yaml
my_controller:
  module: controllerx
  class: CustomLightController
  controller: e1810_controller
  integration: deconz
  light: light.my_light
  mapping:
    1002: sync
```
This controller uses sync when the middle button from the E1810 is pressed. The proposed configuration is:
```yaml
my_controller:
  module: controllerx
  class: CustomLightController
  controller: e1810_controller
  integration: deconz
  light: light.my_light
  mapping:
    1002: 
      action: sync
      brightness: 100
      color_temp: 200
```
In this case, sync will be receiving two parameters, `brightness` and `color_temp`. We will be able to keep using the default action `sync` by referencing as before, but internally it will be calling the sync function with its brightness and color_temp by default.

### Describe alternatives you've considered

In the meantime, we can keep using `CallServiceController` to give the behaviour that we want exactly.

### Additional context

The initials of this feature request come from this issue (#73)

_Note_: This feature request might evolve and change.

**Comment by lensherm** [comment]
Another nice to have predefined action would be 'mute' for media players

**Comment by xaviml** [comment]
Hi @lensherm,

I created the following ticket to add `mute` as a predefined action: https://github.com/xaviml/controllerx/issues/238

Thank you,
Xavi M.

**Comment by lensherm** [comment]
> Hi @lensherm,
> 
> I created the following ticket to add `mute` as a predefined action: #238
> 
> Thank you,
> Xavi M.

@xaviml , thank you! Much appreciated.

**Comment by xaviml** [comment]
This has been released in ControllerX [v4.14.0](https://github.com/xaviml/controllerx/releases/tag/v4.14.0). You can read more about it in the release note.
